aiobotocore/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '2.7.0'
+__version__ = '2.8.0'
```

Comment:
I think we can just do a dot release, no need for major rev

Reply:
@thehesiod updated to 2.7.1 also I don't know what's with coverity, because it was pointing that some parts were not covered while they were. I also updated the unit tests so I'm getting 100% for that file now. Can you run the pipeline again?